bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/api/OpenSprinklerHttpApiV100.java

```diff
@@ -147,27 +155,13 @@ public boolean isStationOpen(int station) throws Exception {
 
     @Override
     public boolean isRainDetected() throws GeneralApiException, CommunicationApiException {
-        String returnContent;
-        int rainBit = -1;
-
         try {
-            returnContent = Http.sendHttpGet(getBaseUrl() + CMD_STATUS_INFO, getRequestRequiredOptions());
+            if (statusInfo().rs == 1) {
+                return true;
+            } else {
+                return false;
+            }
         } catch (Exception exp) {
```

Comment:
do not catch `Exception`

Reply:
Removed the whole try/catch block, this was a missing refactoring from me, the separate thrown exception doesn't give any value :)